helper_functions/sunglare_helperFunctions.js

```diff
@@ -0,0 +1,302 @@
+// *---------------*
+// Sunglare querystrings
+// *---------------*
+
+function GetQueryStrings() {
+    var querystrings = {
+        user: {
+            type: 'string',
+            description: 'The user name.',
+            default: ''
+        },
+        moduleType: {
+            type: 'string',
+            description: 'The type of predictive module.',
+            default: 'sunglare'
+        },
+        sort: {
+            type: 'string',
+            description: 'The sorting method used in the report PDF.',
+            default: 'crash-sort'
+        },
+        startYear: {
+            type: 'string',
+            description: 'The start year for crashes.',
+            default: '2015'
+        },
+        endYear: {
+            type: 'string',
+            description: 'The end year for crashes.',
+            default: '2020'
+        },
+        crashAttributes: {
+            type: 'string',
+            description: 'Comma seperated list of Crash Attribute codes based on the NJTR-1 form.'
+        },
+        travelDirectionCodes: {
+            type: 'string',
+            description: 'Comma seperated list of Travel Direction codes based on the NJTR-1 form.'
+        },
+        timeOfDayCodes: {
+            type: 'string',
+            description: 'Comma seperated list of Time of Day codes based on the NJTR-1 form.'
+        },
+        signalizedIntersectionCodes: {
+            type: 'string',
+            description: 'Comma seperated list of Signalized Intersection codes based on the NJTR-1 form.'
+        },
+        sri: {
+            type: 'string',
+            description: 'SRI code.'
+        },
+        countyCode: {
+            type: 'string',
+            description: 'County Code.'
+        },
+        muniCode: {
+            type: 'string',
+            description: 'Municipality code.'
+        }
+    }
+    return querystrings;
+}
+
+// *---------------*
+// SQL Clause Helper Functions
+// *---------------*
+
+// creates the clause for SRI, county, or muni
+function CreateLocationClause(queryString) {
```

Comment:
this may as well be a constructor (LocationClause instead of CreateLo...)
it makes it easier to abstract later without changing all the reliant code

Reply:
The "create..." functions are one time use. The returning data is a string that is combined with other strings to create the specific sunglare statistics query. It does not hold any other useful data to it. 

Otherwise, our historic use of constructers are objects that hold many attributes and the object persists throughout a user's session.